app/generator/create-log.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../data')
+
+const createLog = async (statementData, filename, dateGenerated) => {
```

Comment:
Idk if we have a semi-pseudo dev practice of calling db save downs as functions with "save" in the name rather than `create`

Reply:
It's just a single insert sql call so would effectively be a single transaction.

I'll rename to save.

Reply:
Renamed